src/tic_tac_toe/ai.clj

```diff
@@ -1,25 +1,41 @@
 (ns tic-tac-toe.ai
   (:require [tic-tac-toe.board :refer [board-size move matrix-convrt empty-space move]]
-            [tic-tac-toe.game :refer  [game-depth  player1-marker player2-marker winner?]]
+            [tic-tac-toe.game :refer  [game-depth winner?]]
              [tic-tac-toe.protocol.player :refer [PlayerProtocol]]))
 
-(defn score-game [board]
-	(cond
-	 (= player2-marker (winner? board)) (- 10 (game-depth board))
-	 (= player1-marker (winner? board)) (- (game-depth board) 10)
-	  :else 0))
+(def marker-1 "x")
+(def marker-2 "o")
+(def place-holder (atom {:player-marker marker-1 :ai-marker marker-2}))
```

Comment:
@Dholness2 You shouldn't use an `atom` for this. You should be able to manage this by passing around a players list/vector that you can cycle through on each play.